run

```diff
@@ -703,6 +741,7 @@ if [ -n "$GDB_ADDRESS" ]; then
     xml::gdb::set "$XML_NAME" "$GDB_ADDRESS"
 fi
 
+while  [ -n "$NODE" ] && [ "$NODE" -ne "0" ] && ! [[ -f $USR_CACHE_DIR/ktest-0-$DISK_OWNER/start.log ]]; do sleep 1; done
```

Comment:
It would hang indefinitely if the disk doesn't come up. Also the dependency on the start log is questionable.

What is ktest-0?